### DISCUSSION TITLE
negative classname - merge and remove

### DISCUSSION THREAD

**Discussion by ackvf** [dcastil/tailwind-merge#390] [Ideas]
Hello 👋, I have a component that I can't control, but it accepts a `labelClassName` prop, which is then used like this within the component:
```jsx
<Typography className={twMerge('mr-8 truncate', labelClassName)} ... />
```

That `truncate` is giving me hard time and I need to remove it. My suggestion would be to use something like this
```
twMerge('truncate', '-truncate') // gives empty result: ''
```
Using "negative" classname would result in it being removed from the result.

**Comment by dcastil** [comment]
Hey @ackvf! 👋

Hmmm that would conflict with negative classes like `-m-3` that are defined in Tailwind CSS. We'd need to differentiate between those which would ultimately lead to a confusing experience.

You could set the value of `labelClassName` to `text-clip`, that should remove the `truncate` class within the component. Is that an option? 🤔

**Reply by ackvf** [reply]
I see, you are completely right. Thank you for your suggestion
